new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Domain.java

```diff
@@ -405,17 +411,26 @@ public static boolean scaleClusterWithWLDF(String clusterName,
           Paths.get(PROJECT_ROOT + "/../src/scripts/scaling/scalingAction.sh"),
           Paths.get(domainHomeLocation + "/bin/scripts/scalingAction.sh"));
     } catch (ApiException apex) {
-      logger.severe("Got ApiException while copying file to admin pod {0}", apex.getResponseBody());
+      logger.severe("Got ApiException while copying file {0} to admin pod {1} in namespace {2}, exception: {3}",
+          PROJECT_ROOT + "/../src/scripts/scaling/scalingAction.sh", adminServerPodName, domainNamespace,
+          apex.getResponseBody());
       return false;
     } catch (IOException ioex) {
-      logger.severe("Got IOException while copying file to admin pod {0}", ioex.getStackTrace());
+      logger.severe("Got IOException while copying file {0} to admin pod {1} in namespace {2}, exception: {3}",
+          PROJECT_ROOT + "/../src/scripts/scaling/scalingAction.sh", adminServerPodName, domainNamespace,
+          ioex.getStackTrace());
       return false;
     }
 
     logger.info("Adding execute mode for scalingAction.sh");
     result = exec(adminPod, null, true,
         "/bin/sh", "-c", "chmod +x " + domainHomeLocation + "/bin/scripts/scalingAction.sh");
     if (result.exitValue() != 0) {
+      logger.info("failed to add execute mode for file {0} in admin server pod {1}",
```

Comment:
modify Kubernetes.exec() method to log the msg, stdout and stderr when exitValue is non-zero, then you don't have to log at the caller

Reply:
fixed